lib/ts/recipe/session/utils.ts

```diff
@@ -154,6 +175,16 @@ export function validateAndNormaliseUserInput(
         }
     }
 
+    if (cookieSameSite === "none" && !enableAntiCsrf) {
```

Comment:
I think this logic is also being checked somewhere else in the code.. we can remove it from that place